src/components/common/Forms/RegisterForm/RegisterForm.jsx

```diff
@@ -39,18 +40,23 @@ export const RegisterForm = () => {
     const [showConfirmPassword, setShowConfirmPassword] = useState(false);
     const [confirmPasswordError, setConfirmPasswordError] = useState('');
     const [isChecked, setIsChecked] = useState(false);
+    const [isMessage, setIsMessage] = useState(false);
```

Comment:
Please better use `isError`, it's more describing what it does

Reply:
has been fixed